<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArpAttacker</class>
 <widget class="QWidget" name="ArpAttacker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>236</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="tip1">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tip2">
      <property name="text">
       <string>14</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tip3">
      <property name="text">
       <string>105</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tip4">
      <property name="text">
       <string>127</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>target host ip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QLabel" name="isSuccessful">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>gateway ip</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>236</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="gip1">
      <property name="text">
       <string>222</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gip2">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gip3">
      <property name="text">
       <string>79</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="gip4">
      <property name="text">
       <string>254</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
